### PR TITLE
Azure snapshot relations

### PR DESCRIPTION
**Pull request by GauravChawda**

### CODE REVIEW

File: cartography/intel/azure/compute.py [hunk 1]
```diff
@@ -103,6 +103,18 @@ def load_vms(neo4j_session: neo4j.Session, subscription_id: str, vm_list: List[D
     MERGE (v)-[rel:HAS]->(i)
     ON CREATE SET rel.firstseen = timestamp()
     SET rel.lastupdated = $update_tag
+    WITH vm, v
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/compute.py:106]: add this query before `MATCH (i:AzureManagedIdentity{id: ua})` statement

File: cartography/intel/azure/compute.py [hunk 2]
```diff
@@ -763,13 +775,36 @@ def sync_disk(
     cleanup_disks(neo4j_session, common_job_parameters)
 
 
+def _attach_snapshot_disk(neo4j_session: neo4j.Session, snapshot_id: str, disk_id: str, update_tag: int):
+    attach_snapshot_disk = """
+    MATCH (s:AzureSnapshot{id: $snapshot_id})
+    WITH s
+    MATCH (d:AzureDisk{id: $disk_id})
+    WITH s, d
+    MERGE (s)-[r:ATTACHED_TO]->(d)
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/compute.py:784]: create relationship like disk ->(HAS)->snapshot

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
few minor changes needed 
also create cleanup jobs for new nodes and relationships

**Review by ganeshjagdale9545 — APPROVED** [review]
Looks good

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**